### ISSUE TITLE
What will be your next big updates or goal?

### ISSUE THREAD

**Issue by Brainshot** [UnamSanctam/SilentCryptoMiner#17]
So what is your next development unam? what are you aiming to add the most

something like for example if the PC cant mine on GPU it will automatically detect and mine XMR on CPU

**Comment by Brainshot** [comment]
> 
> 
> Next I'm going to update both my UnamBinder and UnamDownloader and then continue to work on my other unreleased projects such as the Crypto Clipper, Stealer, Discord Worm and RAT. I'll also work on any features for the miner that I can think of, mainly it's to get the rootkit out of the unstable experimental phase.
> 
> There isn't really any reason to do that, you should optimally build a miner that mines both ETH/ETC and XMR (add two miners in the builder) so that you have both a CPU and GPU miner. The CPU miner barely affects the computer at all and neither the ETH/ETC and XMR miner will interfere with each other.

Interesting will you be adding stealers and rat to the panel? or is it gonna be just other projects that are not added to the miners

**Comment by T3Toxic** [comment]
idc the price of the discord worm but I'll buy it. also, if I add eth and etc on the same miner, it will mine both or will it mine eth when the GPU has enough VRAM?

**Comment by UnamSanctam** [comment]
> Interesting will you be adding stealers and rat to the panel? or is it gonna be just other projects that are not added to the miners

I could add the stealer yes, though adding the RAT wouldn't do much since that would have its own program.

> idc the price of the discord worm but I'll buy it. also, if I add eth and etc on the same miner, it will mine both or will it mine eth when the GPU has enough VRAM?

All my projects have always and will always be free so if I release it then it would be on my GitHub like everything else. If you have both an ETC and ETH miner then it will mine both, but since most GPUs don't have enough VRAM to mine both it will just mine one of them. If you have the ETH above the ETC in the "Miners" list then it will start the ETH before the ETC (though both are started nearly instantly).

**Comment by T3Toxic** [comment]
okay thanks, do you have any pools that you can recommend for etc, ethermine has to have a highest preforming miner's IP to withdraw.

**Comment by UnamSanctam** [comment]
Almost all pools except nanopool does it that way (though nanopool isn't the best either). Usually you mine just yourself to the pool first and change the minimum withdrawal with your IP and then connect everyone else.

**Comment by T3Toxic** [comment]
wait would I give them my ip or would that not work. but do you have any that you would recommend for etc as I legit only know ethermine

**Comment by UnamSanctam** [comment]
Yes but you should always use a VPN or Tor since the pool (and nearly every website) always logs your IP when you visit it normally. The only pool that does not use the IP I know of is nanopool, personally I have been testing ezil.me instead and that has been working well so far.

**Comment by T3Toxic** [comment]
I'm from africa, i don't think i need to worry about the law tbh

**Comment by T3Toxic** [comment]
also my highest worker wont be my IP, would it? as it would have multiple ips if there is different pcs running it

**Comment by UnamSanctam** [comment]
> I'm from africa, i don't think i need to worry about the law tbh

Haha maybe, though using Tor wouldn't hurt while visiting the pool.

> also my highest worker wont be my IP, would it? as it would have multiple ips if there is different pcs running it

If only you mine with your own computer before you create the miner or change them to that pool then only your IP will be there.

**Comment by T3Toxic** [comment]
wait do you meam make the miner on the pool and miner with it for a couple minutes, then my IP would be on it?

**Comment by UnamSanctam** [comment]
Yes, before you make any other people mine on the pool just mine on it yourself so only you would be mining, then you can use your IP to change the minimum payout.

**Comment by Brainshot** [comment]
> 
> 
> Yes, before you make any other people mine on the pool just mine on it yourself so only you would be mining, then you can use your IP to change the minimum payout.

Hey unam the webpanel is still not showing GPU names even though i replaced the dll file

**Comment by T3Toxic** [comment]
> Yes, before you make any other people mine on the pool just mine on it yourself so only you would be mining, then you can use your IP to change the minimum payout.

oh cool, thanks

**Comment by UnamSanctam** [comment]
> > Yes, before you make any other people mine on the pool just mine on it yourself so only you would be mining, then you can use your IP to change the minimum payout.
> 
> Hey unam the webpanel is still not showing GPU names even though i replaced the dll file

Did you replace all the files?

**Comment by Brainshot** [comment]
> 
> 
> > > Yes, before you make any other people mine on the pool just mine on it yourself so only you would be mining, then you can use your IP to change the minimum payout.
> > 
> > 
> > Hey unam the webpanel is still not showing GPU names even though i replaced the dll file
> 
> Did you replace all the files?

no only  unamwebpanel.db

**Comment by UnamSanctam** [comment]
Then of course if won't show in the miner datatable, since it haven't been told to show that. Also replace the `/assets/php/datables.php` and the `/api/endpoint.php` files to make it accept the GPU from the miners and to show it on the datatable.

**Comment by Brainshot** [comment]
> 
> 
> Then of course if won't show in the miner datatable, since it haven't been told to show that. Also replace the `/assets/php/datables.php` and the `/api/endpoint.php` files to make it accept the GPU from the miners and to show it on the datatable.

done added i can see the GPU thing but there is no GPU shown

should i just replace all files?

**Comment by Brainshot** [comment]
nvm it worked now

**Comment by Brainshot** [comment]
> 
> 
> > Interesting will you be adding stealers and rat to the panel? or is it gonna be just other projects that are not added to the miners
> 
> I could add the stealer yes, though adding the RAT wouldn't do much since that would have its own program.
> 
> > idc the price of the discord worm but I'll buy it. also, if I add eth and etc on the same miner, it will mine both or will it mine eth when the GPU has enough VRAM?
> 
> All my projects have always and will always be free so if I release it then it would be on my GitHub like everything else. If you have both an ETC and ETH miner then it will mine both, but since most GPUs don't have enough VRAM to mine both it will just mine one of them. If you have the ETH above the ETC in the "Miners" list then it will start the ETH before the ETC (though both are started nearly instantly).

hey unam i always have been wondering why do you release your stuff? make it all for free? why not just sell it

you would make big profit

**Comment by UnamSanctam** [comment]
I have a real job where I earn money and then I create these in my spare time as a hobby for fun so I don't really want charge money for anything, since then it kind of stops being just a hobby.
